utils/dependency_cross_matcher.py

```diff
@@ -43,14 +43,31 @@ def main(argv=[], prog_name=''):
     parser = argparse.ArgumentParser(prog='DependencyCrossmatcher',
                                      description='Checks the version mismatches for dependencies '
                                                  'in Cargo based repositories')
+    parser.add_argument("-c", "--compare", action='store_true',
+                        help='Check for mismatches between 2 repositories')
     parser.add_argument('--deps_dir',
                         required=True,
-                        help='Existing directory that contains the Cargo.toml for analyzing'
+                        nargs='+',
+                        help='Existing directories that contain Cargo.toml for analyzing '
                              'dependencies')
     args = parser.parse_args()
 
-    mismatches = run_deps_mismatcher(run_cargo_tree(args.deps_dir))
-    print_deps(mismatches)
+    mismatches = dict()
+
+    if args.compare:
+        parsec_repo, parsec_tool_repo = args.deps_dir
+        mismatches_parsec = run_deps_mismatcher(run_cargo_tree(parsec_repo))
+        mismatches_parsec_tool = run_deps_mismatcher(run_cargo_tree(parsec_tool_repo))
```

Comment:
I think the parsec service needs a few additional flags in the cargo tree command right?

Reply:
Done

Reply:
Thanks for noticing! Well spotted